types/gulp-header/tslint.json

```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Remove all the rules here and fix errors if any. new definitions cannot have anything except extends clause here.